src/util/sandbox/fd_sandbox.h

```diff
@@ -64,15 +64,49 @@
       We have the luxury of disallowing all of the syscalls that might
       have received less scrutiny. */
 
+struct seccomp_allowance {
+   /* `branch` specifies which type of allowance the union contains */
+   enum {
```

Comment:
don't use enums (kbowers has a rant on this), should be

FD_SANDBOX_RULE_ALLOW_SYSCALL
FD_SANDBOX_RULE_ALLOW_SYSCALL_ARGUMENTS